### PR TITLE
Update UG and UC for batch-delete

### PR DESCRIPTION
**Pull request by MinkhantEain**
Updated only batch-delete

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,6 +145,18 @@ Examples:
 Alternative Format: `delete NAME`
 * If there are duplicates, a list of relavant entries will be shown. 
 
+### Deleting in a batch : `batch-delete`
+
+Delete all contacts from cher with contains **all** the specified tags.
+
+Format: `batch-delete [t/TAG]...`
```
Comment (yyueyii) [inline docs/UserGuide.md:152]: Items in square brackets are optional, but here tags shldn't be optional

File: docs/UserGuide.md [hunk 2]
```diff
@@ -197,6 +209,7 @@ Action | Format, Examples
 **Add** | `add n/NAME r/ROLE p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho r/student p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Secondary 1`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
+**Batch-Delete**| `batch-delete [t/TAG]...`<br> e.g. `batch-delete t/friends t/colleagues t/owesmoney t/...`
```
Comment (yyueyii) [inline docs/UserGuide.md:212]: Here as well

**Review by yyueyii — DISMISSED** [review]
Some minor changes, otherwise lgtm!

File: docs/images/BatchDeleteExample.png
Comment (yyueyii) [inline docs/images/BatchDeleteExample.png:1]: I'm not sure how this image shows batch delete

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -390,6 +390,17 @@ Priorities: High (Must-Have), Medium (Nice-to-Have), Low (Could-Have), Trivial (
 2. Cher saves contacts in memory into local file. <br>
    Use case ends.
 
+#### Use case: UC5 - Batch delete
+**MSS**
+1. User enters a command to delete all contacts with specific tags.
+2. Cher will remove all contacts containing specified tags.
+3. Cher will show the contacts that have been removed.
+
```
Comment (zhou-colla) [inline docs/DeveloperGuide.md:398]: Should we put in "Use case ends" here?

**Review by zhou-colla — COMMENTED** [review]
Other than the missing use case ends, everything looks good!

**Review by zhou-colla — APPROVED** [review]
Looks good!

**Pull request merged**